### PR TITLE
Fix crash loading style from postgres db when there is no "layer_styl…

### PR DESCRIPTION
**Pull request by elpaso**
…es" table in the database

### CODE REVIEW

File: src/app/qgsvectorlayerloadstyledialog.cpp [hunk 1]
```diff
@@ -141,24 +142,27 @@ void QgsVectorLayerLoadStyleDialog::initializeLists( const QStringList &ids, con
   mRelatedTable->setHorizontalHeaderLabels( relatedTableHeader.split( ';' ) );
   mOthersTable->setHorizontalHeaderLabels( othersTableHeader.split( ';' ) );
   mRelatedTable->setRowCount( sectionLimit );
-  mOthersTable->setRowCount( ids.count() - sectionLimit );
+  mOthersTable->setRowCount( sectionLimit >= 0 ? ( ids.count() - sectionLimit ) : 0 );
   mRelatedTable->setDisabled( relatedTableNOfCols == 1 );
   mOthersTable->setDisabled( othersTableNOfCols == 1 );
 
-  for ( int i = 0; i < sectionLimit; i++ )
+  if ( sectionLimit > 0 )
```
Comment (3nids) [inline src/app/qgsvectorlayerloadstyledialog.cpp]: this is redundant, if sectionLimit = 0, nothing will be done by the for loop
do you prefer to keep it?

Comment (elpaso) [reply src/app/qgsvectorlayerloadstyledialog.cpp]: Look at the second loop, that is the one that crashed when sectionlimit was -1

Comment (3nids) [reply src/app/qgsvectorlayerloadstyledialog.cpp]: right, sorry!

Comment (elpaso) [reply src/app/qgsvectorlayerloadstyledialog.cpp]: np, thanks for reviewing!

**Pull request merged**